Client/Frontend/Widgets/PhotonActionSheet.swift

```diff
@@ -273,7 +273,7 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: PhotonActionSheetUX.CellName, for: indexPath) as! PhotonActionSheetCell
         let action = actions[indexPath.section][indexPath.row]
-        
+        cell.accessibilityIdentifier = action.iconString
```

Comment:
Just landed this today. You wont need to do this anymore. 
https://github.com/mozilla-mobile/firefox-ios/commit/c17db926ecb38c82658845ebcce68494da1ede0d